spec/helpers/zaf.js

```diff
@@ -0,0 +1,22 @@
+function appendParams(url, params) {
```

Comment:
I'd have implemented this by passing in `window.location.search` splitting into an array on `&` and rejoining once new params are added, then you can just append `window.location.hash` on the end and `${window.location.origin}${window.location.pathname}` on the front. I'm not sure which would be more readable, my suggestion would probably be more verbose.